streams/runners/storm/runtime/src/main/java/com/hortonworks/streamline/streams/runtime/storm/spout/AvroStreamsSnapshotDeserializer.java

```diff
@@ -65,7 +65,10 @@ private Object convertValue(Object deserializedObj) {
             List<Schema.Field> fields = indexedRecord.getSchema().getFields();
             ImmutableMap.Builder<String, Object> keyValues = ImmutableMap.builder();
             for (Schema.Field field : fields) {
-                keyValues.put(field.name(), convertValue(indexedRecord.get(field.pos())));
+                Object currentValue = convertValue(indexedRecord.get(field.pos()));
+                if (currentValue != null) {
```

Comment:
@harshach Its not null message from kafka at this point. Its a nullable field from avro schema inside the message. We dont have kafka offset at this point in code. This is called after the deserializer.deserialize on kafka has been called for value in kafka payload. That deserialized value is only passed here. If we want to log offset, i will have to change around signatures of methods to pass offset information as well and log it. We can log that we did not receive a value for nullable avro field here. But I think we can probably not do that as well since user should be aware of avro schema. Do you still prefer adding any debug log here?